applications/plugins/CollisionOBBCapsule/examples/CollisionGroups.scn

```diff
@@ -7,7 +7,7 @@ Here, the collision models of the 3 cubes are defined in the group 1. Therefore,
 in the group 1, therefore cubes collide with it.
```

Comment:
There is ```examples/Benchmark/Performance/CollisionGroupManager/2systems_with_CollisionGroupManager.scn``` that is supposed to use collision groups [https://github.com/sofa-framework/sofa/blob/master/examples/Benchmark/Performance/CollisionGroupManager/2systems_with_CollisionGroupManager.scn](https://github.com/sofa-framework/sofa/blob/master/examples/Benchmark/Performance/CollisionGroupManager/2systems_with_CollisionGroupManager.scn).

Is it enough or shall I create an issue indicating to create a similar example but without the plugin (if possible because IDK what this plugin is actually doing)?